backend/db/migrations/20180329230713-create-show.js

```diff
@@ -37,6 +37,11 @@ export function up (queryInterface, Sequelize) {
       type: Sequelize.DATE,
       allowNull: false
     },
+    finalized: {
```

Comment:
Don't update the timestamp on this file. You break migrations since there's now a `user_shows` migration before a `shows` migration.

Reply:
Ideally this would have been a new migration w/ [`addColumn`](http://docs.sequelizejs.com/class/lib/query-interface.js~QueryInterface.html#instance-method-addColumn), but whatever...

Reply:
yeah, I know, but we've yet to do that. The process we've had since we started was to just update. I can go back to the old name and do the new add column thing.